docs/joss_paper/paper.md

```diff
@@ -50,72 +62,72 @@ affiliations:
+If images are available in pairs, two dataset loaders are currently available for paired images with- and without labels. Many clinical applications for tracking organ motion and other temporal changes require _intra-subject_ _single-modality_ image registration. Registering lung CT images for the same patient, acquired at expiratory and inspiratory phases [@hering_alessa_2020_3835682], is such an example of both unsupervised (without labels) and combined supervision (trained with label dissimilarity based on segmentation of the anatomical structures). Furthermore, registering prostate MR, acquired before surgery, and the intra-operative transrectal ultrasound images is an example of weakly-supervised training for multimodal image registration [@hu2018weakly]. Another DeepReg Demo illustrates MR-to-ultrasound image registration is to track tissue deformation and brain tumour resection during neurosurgery [@xiao2017resect], in which combining unsupervised and weakly-supervised losses may be useful.
 
 ## Unpaired images
-Unpaired images are found in applications such as _single-modality_ _inter-subject_ registration, with two dataset loaders for labelled and unlabeled image data. Registering different MR images of the brain from different subjects has played a fundamental role in population studies. Two applications which register unpaired inter-subject CT images, for lung and abdominal organs, represent the type of registration that may benefit from added label supervision. Additionally, an example which demonstrates the support for experiments such as cross-validation in `DeepReg` has been included in DeepReg Demos. This example registers 3D ultrasound images from different surgical patients.
+Unpaired images are found in applications such as _single-modality_ _inter-subject_ registration, with two dataset loaders for labelled and unlabeled image data. Registering different MR images of the brain from different subjects [@simpson2019large] has played a fundamental role in population studies. Two applications which register unpaired inter-subject CT images, for lung [@hering_alessa_2020_3835682] and abdominal organs [@adrian_dalca_2020_3715652], respectively, represent the type of registration that may benefit from added label supervision. Additionally, an example which demonstrates the support for experiments such as cross-validation in DeepReg has been included in DeepReg Demos. This example registers 3D ultrasound images from different prostate cancer patients during surgical cases.
```

Comment:
Line 1/2: need consistency in `labeled - unlabeled' (either `ll' or `l' in both)